src/main/java/com/google/research/bleth/simulator/AbstractSimulation.java

```diff
@@ -18,9 +18,11 @@
 import static com.google.common.collect.ImmutableMap.toImmutableMap;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMultimap;
 import com.google.common.collect.ImmutableTable;
 import com.google.common.collect.LinkedListMultimap;
 import com.google.common.collect.Table;
+
```

Comment:
Google recommended style does not separate java.* imports from other imports. https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

(Applies to other files touched in this PR as well)

Reply:
As discussed, will be fixed in a separate PR.